lib/db.js

```diff
@@ -476,43 +467,43 @@ DB.prototype._createTable = function (keyspace, schema, tableName) {
     });
 
     // Finally, create the main data table
-    var cql = 'create table if not exists '
+    var sql = 'create table if not exists '
         + dbu.cassID(keyspace + '_' + tableName) + ' (';
     for (var attr in schema.attributes) {
         var type = schema.attributes[attr];
-        cql += dbu.cassID(attr) + ' ';
+        sql += dbu.cassID(attr) + ' ';
         switch (type) {
```

Comment:
Well, IMHO, there should be an order to things. I fail to see how relevant is getting the test for "insert if not exists" to pass if the underlying table hasn't been created properly. Creating tables and indices is crucial to getting everything else right.